hmf-common/src/main/java/com/hartwig/hmftools/common/linx/LinxDataLoader.java

```diff
@@ -240,8 +247,69 @@ private static Map<Integer, Integer> loadSvToCluster(@NotNull String filename) t
             {
                 svToCluster.put(svId, clusterId);
             }
+
+            if(clusterIdToChainCount.containsKey(clusterId))
+            {
+                clusterIdToChainCount.put(clusterId, clusterIdToChainCount.get(clusterId) + 1);
```

Comment:
To double-check: Every line in the SV data for the same cluster represents an extra chain link?

Reply:
That was my understanding, and my earlier testing seemed to work. SvVisualizer calls these records clusterLinks, my naming here is maybe not great.